src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -125,7 +122,7 @@ public static function defaultSettings() {
     return parent::defaultSettings() + [
       'iiif_group' => TRUE,
       'mediasource' => 'json_key',
-      'json_key_source' => 'as:image',
+      'json_key_source' => 'as:document',
```

Comment:
This is the part I'm a bit more nervous about, not sure if the as:image was a leftover or it is intentional as I've seen the other formatters rely heavily on the key source to get the file from the metadata in json

Reply:
Sorry for the late response here: Bookreader (paged) can work on both, images and Documents, so please leave image as default. We ship with examples of this and we have a few in play.archipelago.nyc too

Reply:
Ok, I retract my comment. This is a left over (I can't see track of all my own code apparently). We do not need that value anymore. Issue is originally we have 4 strategies. (still in the comments), as:image, as:document, metadata display and external URL IIIF manifest.

Since this one came https://github.com/esmero/format_strawberryfield/pull/29 in (and that one, please look at it) was more like a rescue operation after a too large pull got merged (🤕 ) with bugs and I had to fix it ASAP we have now only 2 operations.

Which also means... since we are only using the "twig template" or the external URL... gosh... we only need to use the actual per NODE cache here, since all is driven by what the JSON provides. If a file gets changed/removed its also about the JSON itself that gets updated. 

Conclusion:
1.- YES, good catch and good you were nervous: we do not need/nor use that Key anymore. Means the form needs to be gone, the setting and even the schema? (schema changes are a mess.. so good with making that change too? I guess it will also imply a hook update to remove the setting)
2.- All your other changes are great but we can't use File Cache here at all. I mean, it is the same as the JSON cache which is the same as the NODE cache so its redundant (sorry)

and
protected function getFile(array $mediaitem, string $type)  (which is a great function) should go to the parent class so we can use it where its really needed, e.g the Media Formatter - Open Seadragon (which uses directly media) or to this new class, which will be used for anything where access to media is dynamic (now only WARC) and https://github.com/esmero/format_strawberryfield/blob/1.0.0-RC2/src/Plugin/Field/FieldFormatter/StrawberryDirectJsonFormatter.php . Up to you

Reply:
Should we just close this PR and open separate issues for the components that are actually useful then?

Reply:
Ok, it may be the best. Let's gather and recover energy and take over smaller chunks of this. Sorry! But it was a nice discussions and we learned so much!